project/Dependencies.scala

```diff
@@ -15,44 +15,30 @@ import Keys._
 
 object Dependencies {
   val resolutionRepos = Seq(
-    "ScalaTools snapshots at Sonatype" at "https://oss.sonatype.org/content/repositories/snapshots/",
-    "Twitter Maven Repo" at "http://maven.twttr.com/" // For Twitter's util functions
+    "ScalaTools snapshots at Sonatype" at "https://oss.sonatype.org/content/repositories/snapshots/"
   )
 
   object V {
-    val maxmind = "1.2.11"
-    object collUtils {
-      val _29   = "5.3.10"
-      val _210  = "6.3.4"
-      val _211  = "6.20.0"     
-    }
-    object specs2 {
-      val _29   = "1.12.4.1"
-      val _210  = "1.14"
-      val _211  = "2.3.13"
-    }
+    val maxMind = "1.2.11"
+    val specs2  = "3.9.5"
   }
 
   object Libraries {
-    val maxmind = "com.maxmind.geoip"          %  "geoip-api"            % V.maxmind
+    val maxMind = "com.maxmind.geoip" % "geoip-api" % V.maxMind
     object collUtils {
-      val _29   = "com.twitter"                %  "util-collection"      % V.collUtils._29
-      val _210  = "com.twitter"                %% "util-collection"      % V.collUtils._210
-      val _211  = "com.twitter"                %% "util-collection"      % V.collUtils._211
-    }
-    object specs2 {
-      val _29   = "org.specs2"                 %% "specs2"               % V.specs2._29        % "test"
-      val _210  = "org.specs2"                 %% "specs2"               % V.specs2._210       % "test"
-      val _211  = "org.specs2"                 %% "specs2"               % V.specs2._211       % "test"
+      val all  = "com.google.guava" % "guava" % "18.0"
     }
+    val specs2  = Seq(
+      "org.specs2" %% "specs2-core"  % V.specs2 % Test,
+      "org.specs2" %% "specs2-junit" % V.specs2 % Test,
+      "org.specs2" %% "specs2-mock"  % V.specs2 % Test
+    )
   }
 
-  def onVersion[A](all: Seq[A] = Seq(), on29: => Seq[A] = Seq(), on210: => Seq[A] = Seq(), on211: => Seq[A] = Seq()) =
-    scalaVersion(v => all ++ (if (v.contains("2.9.")) {
-      on29
-    } else if (v.contains("2.10.")) {
-      on210
-    } else {
+  def onVersion[A](all: Seq[A] = Nil, on211: => Seq[A] = Nil, on212: => Seq[A] = Nil) =
```

Comment:
I don't think this is needed anymore

Reply:
I [forked the project](https://github.com/mslinn/scala-maxmind-iplookups/tree/0.4.0) to use build.sbt instead of project/*.scala, and also published to bintray so I can use the project right away. I'll decide on keeping the fork or folding it in, depending on whether this PR goes through quickly or not.